KomodoPlatform/atomicDEX-Desktop
# Update API

This branch is for updating 0.5.7.x to 0.5.8 once the required methods and changes have been implemented for compatibility with the v1.0.x-beta API. 

Closes:
- https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/2109
- https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/2046

Done:
- ZHTLC method names and request/response structures updated
- Resolved a P0 orderbook/bestorders bug which caused crashes due to duplicated uuids in response for segwit coins.
- Added Tendermint activation
- Added memo sending for ZHTLC and Tendermint coins

To Test:
- Activate ZOMBIE, use faucet, send funds, do a swap
- Activate IRIS, use faucet, send funds with memo (confirm memo on explorer), do a swap
- Activate ARRR and other tendermint assets
- Try to crash the pro view orderbook (click on things like a mad man)

You likely will need to do a wallet reset to see IRIS and other new coins.
